packages/kbn-elastic-agent-utils/src/agent_guards.ts

```diff
@@ -16,11 +16,17 @@ import type {
 } from './agent_names';
 
 export function isOpenTelemetryAgentName(agentName: string): agentName is OpenTelemetryAgentName {
-  return OPEN_TELEMETRY_AGENT_NAMES.includes(agentName as OpenTelemetryAgentName);
+  return (
```

Comment:
@SylvainJuge there's some test here `packages/kbn-elastic-agent-utils/src/agent_guards.test.ts` do you mind add one to cover the new case, please?

Reply:
Sure I'll try to do that tomorrow!

Reply:
Tests added in caaa564e6b6bc4596b7b2cbae984a3c9c9650ac4